modules/cnv-supported-virtio-drivers.adoc

```diff
@@ -1,32 +1,53 @@
 // Module included in the following assemblies:
 //
+<<<<<<< HEAD
```

Comment:
You need to fix these merge conflict remnants. I'm not reviewing this content because I don't know what you're keeping.

Reply:
@bgaydosrh if this module is not included in any assembly, you should remove this.

Reply:
@bgaydosrh - There's no reason for this module to be included in this PR.

Reply:
Hi Vikram -

At the moment I am unsure how to explicitly remove a file with the old git version I am using (1.8). In this version you must explicitly tell git to forget about the file with "git add <filename>" after removing it from your local branch. Previously I was using the -D argument but believe that now to be bad practice (nothing else was working at the time). Should I try with -d argument, so "git add modules/cnv-supported-virtio-drivers.adoc -d"?